.github/workflows/snapshot-release.yml

```
@@ -15,12 +15,21 @@ jobs:
     if: ${{ github.repository_owner == 'withastro' && github.event.issue.pull_request && startsWith(github.event.comment.body, '!preview') }}
     runs-on: ubuntu-latest
     steps:
-      - name: 'Check if user has admin access (only admins can publish snapshot releases).'
-        uses: 'lannonbr/repo-permission-check-action@2.0.2'
+      - name: Check if user has admin access (only admins can publish snapshot releases).
+        id: checkAccess
+        uses: actions-cool/check-user-permission@v2.2.1
```

Should we use 
```suggestion
        uses: actions-cool/check-user-permission@v2
```

instead so we can get patch updates when available?

---

Ah good catch!